Content.Client/Standing/LayingDownSystem.cs

```diff
@@ -48,6 +51,29 @@ private void OnMovementInput(EntityUid uid, LayingDownComponent component, MoveE
         sprite.Rotation = Angle.FromDegrees(90);
     }
 
+    private void OnDowned(DrawDownedEvent args)
+    {
+        var uid = GetEntity(args.Uid);
+
+        if (!TryComp<SpriteComponent>(uid, out var sprite) 
+            || !TryComp<LayingDownComponent>(uid, out var component))
+            return;
+
+        if (!component.OriginalDrawDepth.HasValue)
+            component.OriginalDrawDepth = sprite.DrawDepth;
+
+        sprite.DrawDepth = (int) DrawDepth.SmallMobs;
```

Comment:
Unable to make a proper test but, make sure that we cannot crawl under tables...
Wait no this change sprite, not fixtures.

Carry on!